x-pack/plugins/ingest_manager/server/services/epm/registry/index.ts

```diff
@@ -47,6 +48,10 @@ export async function fetchList(params?: SearchParams): Promise<RegistrySearchRe
       url.searchParams.set('experimental', params.experimental.toString());
     }
   }
+  const kibanaVersion = appContextService.getKibanaVersion().split('-')[0]; // may be 8.0.0-SNAPSHOT
```

Comment:
@skh Did you try if sending `8.0.0-SNAPSHOT` works? I wonder if the registry should support this somehow?